dotcom-rendering/src/components/Discussion/Comment.tsx

```diff
@@ -318,6 +317,17 @@ export const Comment = ({
 	const [showAbuseReportForm, setAbuseReportForm] = useState(false);
 	const toggleSetShowForm = () => setAbuseReportForm(!showAbuseReportForm);
 
+	const dispatch = useContext(DispatchContext);
+
+	const setPickError = (error: string) => {
+		if (dispatch) {
+			dispatch({
+				type: 'setPickError',
+				error,
+			});
+		}
```

Comment:
The change to `createContext` above means this can be:

```suggestion
		dispatch({
			type: 'setPickError',
			error,
		});
```